src/data_manager/data_file_manager.py

```diff
@@ -40,23 +46,25 @@ def __init__(self, config_file_loc):
         http = urllib3.PoolManager()
 
         # use the recently created snapshot
-        api_url = context_info.env["FMS_API_URL"] + '/api/snapshot/release/' + context_info.env["ALLIANCE_RELEASE"]
-        logger.info(api_url)
+        api_url = '/'.join([context_info.env["FMS_API_URL"],
```

Comment:
I don't think this makes the code more readable, even though the long line is reduced.

Reply:
I can see how it is at best a minimal improvement. I think the most proper way to do this is to use urlparse.join. But then I don't think it would add much improvement. I will change this line back.